.github/scripts/requirements.txt

```diff
@@ -1,4 +1,4 @@
-fab-classic==1.19.1
+fab-classic==1.19.2
```

Comment:
@abejgonzalez why do we have this separate requirements.txt?  Is this vestigial and can it be removed now?  It would be good for this file to have a comment explaining what requirements are kept here and why.  I suspect this is only stuff needed to bootstrap something running `machine_launch_script.sh` and that it is a very minimal set of things needed by a single script.

@t14916 & @sifive-benjamin-morse 👀

Reply:
Correct. In order to spawn an AWS instance manager, we need to first boot into a GH managed VM, install the Fabric/AWS deps, then launch the manager.

https://github.com/firesim/firesim/blob/38a02b3b9083a6c98d1afef75c7b64d4a7bab22c/.github/workflows/firesim-run-tests.yml#L103